.github/workflows/generate-docs.yml

```
@@ -0,0 +1,12 @@
+name: 'generate dbt docs'
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  generate-docs:
+    if: github.event.label.name == 'docs:ready'
+    uses: fivetran/dbt_package_automations/.github/workflows/generate-docs.yml@main
+    with:
+      schema_var_name: recharge_source_integration_tests_05
```

This is the name of the var used to set the schema.
```suggestion
      schema_var_name: recharge_schema
```